l1-contracts/test/fees/FeeRollup.t.sol

```diff
@@ -99,34 +101,14 @@ contract FeeRollupTest is FeeModelTestPoints, DecoderBase {
     vm.fee(l1Metadata[0].base_fee);
     vm.blobBaseFee(l1Metadata[0].blob_fee);
 
-    asset = new TestERC20("test", "TEST", address(this));
-
-    Registry registry = new Registry(address(this), asset);
-    rewardDistributor = RewardDistributor(address(registry.getRewardDistributor()));
-
-    rollup = new Rollup(
-      asset,
-      IRewardDistributor(address(rewardDistributor)),
-      asset,
-      address(this),
-      TestConstants.getGenesisState(),
-      RollupConfigInput({
-        aztecSlotDuration: SLOT_DURATION,
-        aztecEpochDuration: EPOCH_DURATION,
-        targetCommitteeSize: 48,
-        aztecProofSubmissionWindow: EPOCH_DURATION * 2 - 1,
-        minimumStake: TestConstants.AZTEC_MINIMUM_STAKE,
-        slashingQuorum: TestConstants.AZTEC_SLASHING_QUORUM,
-        slashingRoundSize: TestConstants.AZTEC_SLASHING_ROUND_SIZE,
-        manaTarget: MANA_TARGET,
-        provingCostPerMana: provingCost
-      })
-    );
-
-    registry.addRollup(IRollup(address(rollup)));
+    RollupBuilder builder = new RollupBuilder(address(this)).setProvingCostPerMana(provingCost)
```

Comment:
surprised the formatter does not newline each dot here

Reply:
Yes, same, but 🤷